astro.config.mjs

```diff
@@ -23,6 +23,11 @@ export default defineConfig({
             title: "BibleQuiz.com",
             description: "Home of AG Bible Quiz",
             social: [
+                {
+                    icon: "npm",
+                    label: "Apps",
+                    href: "apps",
```

Comment:
This doesn't work when I run locally. I don't see the purpose of this as there are links for the Apps on the home page.